Adding new MCP Section (#131)
kluster-ai/docs: get-started/mcp/tools.md

---
title: Tools reference
description: Reference guide for kluster.ai's MCP verification tools - verify claims and documents with detailed parameters and response formats.
---

# Tools reference

> The [kluster.ai's](https://www.kluster.ai/){target=\_blank} MCP server provides two verification tools that work identically whether deployed [self-hosted](/get-started/mcp/self-hosted/) or via [Cloud MCP](/get-started/mcp/cloud/platform/). These tools enable real-time reliability verification directly within your AI assistant conversations.

Reviewer: ```suggestion
The [kluster.ai's](https://www.kluster.ai/){target=\_blank} MCP server provides two verification tools that work identically whether deployed [self-hosted](/get-started/mcp/self-hosted/){target=\_blank} or via [Cloud MCP](/get-started/mcp/cloud/platform/){target=\_blank}. These tools enable real-time reliability verification directly within your AI assistant conversations.
```

This page documents the tool parameters and response formats you'll see when usisng these tools in any MCP-compatible client.

## Tool overview

> | Tool | Purpose | Best For |
> |:---|:---|:---|
> | `verify` | Verify standalone claims | General statements, trivia, current events, news |
> | `verify_document` | Verify claims about documents | Quotes, data extraction, RAG hallucination checking |

Reviewer: ```suggestion

The following tools are available through the kluster.ai MCP server:

| Tool | Purpose | Best For |
|:---|:---|:---|
| `verify` | Verify standalone claims | General statements, trivia, current events, news |
| `verify_document` | Verify claims about documents | Quotes, data extraction, RAG hallucination checking |
```

### Verify

> Verifies any statement against reliable online sources.

Reviewer: ```suggestion
The verify tool allows you to check any statement against reliable online sources.
```

**Parameters:**

| Parameter | Type | Required | Description |
|:---|:---|:---|:---|
| `claim` | string | Yes | The statement to verify |
| `returnSearchResults` | boolean | No | Include source citations (default: true) |

### Verify document

> Verifies if claims accurately reflect uploaded document content.

Reviewer: ```suggestion
The verify document tool checks that claims accurately reflect the content of the uploaded document.
```

**Parameters:**

| Parameter | Type | Required | Description |
|:---|:---|:---|:---|
| `claim` | string | Yes | Your interpretation of the document |
| `documentContent` | string | Yes | Full document text (auto-provided by MCP client) |
| `returnSearchResults` | boolean | No | Include external sources (default: true) |

## Response fields

All verification tools return the same response structure:

- **`claim`**: The verified claim.
- **`is_accurate`**: Boolean indicating if claim is accurate.
- **`explanation`**: Detailed reasoning for the verdict.
- **`confidence`**: Token usage statistics.
- **`search_results`**: Source citations (if requested).

> **Response format example:**

Reviewer: ```suggestion
An example can be seen below:
```
